ndcube/utils/wcs.py

```diff
@@ -418,9 +418,9 @@ def array_indicies_for_world_objects(wcs, axes=None):
         For each world object, a tuple of array axes identified by their number.
     """
     if axes:
-        world_indices = calculate_world_indices_from_axes(wcs, axes)
+        calculate_world_indices_from_axes(wcs, axes)
```

Comment:
Why is the variable name deleted but not the function?  Should this entire `if...else...` not be deleted if nothing is being set?